card/mod/03_machines/set/type/coffee_script.rb

```diff
@@ -57,5 +57,5 @@ def chunk_list  #turn off autodetection of uri's
 end
 
 def diff_args
-   {:format=>:text}
+   {format: :text}
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.